Packs/CommonScripts/Scripts/GetUserData/GetUserData.py

```diff
@@ -0,0 +1,570 @@
+        okta_get_user_command = Command(
+            name="okta-get-user",
+            brand="Okta v2",
+            args={
+                "userId": user_id,
+                "username": user_name,
+            },
+            output_key="Account",
+            output_function=lambda entry_context: create_account(
+                id=entry_context.get("ID"),
+                username=entry_context.get("Username"),
+                display_name=entry_context.get("DisplayName"),
+                email_address=entry_context.get("Email"),
+                is_enabled=entry_context.get("Status"),
+            ),
+        )
+        aws_iam_get_user_command = Command(
+            name="aws-iam-get-user",
+            brand="AWS - IAM",
+            args={
+                "userName": user_name,
+            },
+            output_key="AWS.IAM.Users",
+            output_function=lambda entry_context: create_account(
+                id=entry_context.get("UserId"),
+                username=entry_context.get("UserName"),
+            ),
+        )
+        msgraph_user_get_command = Command(
+            name="msgraph-user-get",
+            brand="Microsoft Graph User",
+            args={
+                "user": user_id,
+            },
+            output_key="Account",
+            output_function=lambda entry_context: create_account(
+                id=entry_context.get("ID"),
+                username=entry_context.get("Username"),
+                display_name=entry_context.get("DisplayName"),
+                email_address=entry_context.get("Email", {}).get("Address"),
+                job_title=entry_context.get("JobTitle"),
+                office=entry_context.get("Office"),
+                telephone_number=entry_context.get("TelephoneNumber"),
+                type=entry_context.get("Type"),
+            ),
+        )
+        xdr_list_risky_users_command = Command(
+            name="xdr-list-risky-users",
+            brand="Cortex XDR - IR",
+            args={"user_id": user_id},
+            output_key="PaloAltoNetworksXDR.RiskyUser",
+            output_function=lambda entry_context: create_account(
+                id=entry_context.get("id"),
+                type=entry_context.get("type"),
+            ),
+        )
+        iam_get_user_command = Command(
+            name="iam-get-user",
+            args={
+                "user-profile": {"id": user_id}
+                if user_id
+                else {"email": f"{user_name}@{domain}"}
+                if domain and user_name
+                else {"username": user_name}
+                if user_name
+                else {"email": user_email}
+                if user_email
+                else None,
+            },
+            is_generic=True,
+            output_key="Account",
+            output_function=lambda entry_context: create_account(
+                id=entry_context.get("id"),
+                type=entry_context.get("type"),
+            ),
+        )
```

Comment:
I would move all this command creations to another function,
so the main would be easier and shorter to read